index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <div>
+        <h1 id="title">Microverse Survey Form</h1>
+        <p id="description">
+            Thank you for taking the time to help us improve our skills
+        </p>
+    </div>
```

Comment:
Secondly
Please I noticed in the first part of your code you use a div tag instead of a section tag . use section tag because it grouped the generic block of related contents, as these makes our code more organized and easy to understand by fellow developers.